src/java/org/apache/cassandra/cql3/statements/DeleteStatement.java

```diff
@@ -145,8 +145,6 @@ protected ModificationStatement prepareInternal(TableMetadata metadata,
                                                         Conditions conditions,
                                                         Attributes attrs)
         {
-            checkFalse(metadata.isVirtual(), "Virtual tables don't support DELETE statements");
```

Comment:
I like how little it  took 😄

Reply:
Yeah! The most complicated part was just to cover everything with tests :smile: